aries_vcx_core/src/errors/mod.rs

```diff
@@ -7,5 +7,5 @@ mod mapping_indyvdr;
 #[cfg(feature = "vdr_proxy_ledger")]
 mod mapping_indyvdr_proxy;
 mod mapping_others;
-#[cfg(feature = "vdrtools")]
+#[cfg(any(feature = "vdrtools", feature = "modular_libs", feature = "vdr_proxy_ledger"))]
```

Comment:
isn't it redundant? Can there be case that we build without any of these?

Reply:
It is more to clarify that one of these must be enabled to even build the module (regardless we have a "convention" of always building with one of them).